apps/web/src/auth/useCreateAuthContext.ts

```diff
@@ -34,16 +33,6 @@ function saveToken(token: string | null) {
   }
 }
 
-export function getToken(): string {
-  return localStorage.getItem(LOCAL_STORAGE_AUTH_TOKEN_KEY) || '';
-}
-
-export function getTokenClaims(): IJwtClaims | null {
-  const token = getToken();
-
-  return token ? jwtDecode<IJwtClaims>(token) : null;
-}
-
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** ‏ Nice job moving helpers to more appropriate places <3